libs/template/templates/default-python/template/{{.project_name}}/resources/{{.project_name}}_job.yml.tmpl

```diff
@@ -1,10 +1,17 @@
 # The main job for {{.project_name}}
+
+{{- /* Clarify what this job is for for DLT-only users. */}}
```

Comment:
Seems to be an internal-only comment?

Reply:
Yeah, this is a comment only seen by template developers, it's not materialized in the generated code.